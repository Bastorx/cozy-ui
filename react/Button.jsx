import cx from 'classnames'

export default function (props) {
  const { theme, className, children, onClick } = props
  return <button
    role='button'
    className={cx('coz-btn', theme ? `coz-btn--${theme}` : null, className)}
    onClick={onClick}>
    { children }
  </button>
}
