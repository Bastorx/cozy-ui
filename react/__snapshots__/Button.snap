// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button should render examples 1`] = `
"<div>
  <div>
    <div>
      <button aria-busy=\\"false\\" type=\\"submit\\" role=\\"button\\" class=\\"c-btn c-btn--danger\\"><span><svg class=\\"icon\\" width=\\"16\\" height=\\"16\\" style=\\"fill: yellow;\\"><use xlink:href=\\"test-file-stub\\"></use></svg><span>delete</span></span>
      </button>
    </div>
  </div>
</div>"
`;
